Makefile

```diff
@@ -68,6 +68,9 @@ metering-src-docker-build: Dockerfile.src
 metering-operator-docker-build: $(METERING_OPERATOR_DOCKERFILE)
 	docker build -f $< -t $(METERING_OPERATOR_IMAGE_REPO):$(METERING_OPERATOR_IMAGE_TAG) $(ROOT_DIR)
 
+metering-ansible-operator-docker-build: $(METERING_OPERATOR_DOCKERFILE)
```

Comment:
You need add new variables instead of using the existing ones, add them after line 23 https://github.com/operator-framework/operator-metering/pull/701/files#diff-b67911656ef5d18c4ae36cb6741b7965R24

They should have a `METERING_ANSIBLE_OPERATOR_` prefix.
Then you to change your usage of the `METERING_OPERATOR_` variables to use those new variables.

`METERING_ANSIBLE_OPERATOR_IMAGE_REPO` should match the image repo used in the deployment, and `METERING_ANSIBLE_OPERATOR_IMAGE_TAG` should be 4.1 like the other variables.

Reply:
All fixed in 49fcba2dea48345526d0b1508dd7444d413cab7a